config.py

```diff
@@ -3,24 +3,28 @@
 TIMEOUT = 10
 
```

Comment:
and please, think about one constant - one value, do not repeat values with different names of constants

Reply:
I use 2 types of wait:
Timeout - wait something no more then this time. For example wait when some element appears on page.
Pause - wait no matter what. For example - wait some time before checking email on emal server

Reply:
I did not want to say something about `timeout`. In this file`config.py` you use a lot of different constants with the same values